.processes/release.md

```diff
@@ -144,37 +144,49 @@ particular branch to deploy on every change.
+   rm packages/ethereum/deployments/${RELEASE_NAME}/xdai/HoprChannels.json
+   ```
+
+   NOTE: Don't include the deployment of HoprChannels, because this will be re-deployed anyway by the CD system.
+
+   Changes should be committed locally.
+
+1. (on `release/${RELEASE_NAME}`) Now everything is ready and can be pushed to Github: `git push origin`. Wait until the deployment of the basic cluster has completed by the CD.
+1. Create a release tracking PR which can be used to follow CD builds. However, the PR should never be merged! As a reference take a look at https://github.com/hoprnet/hoprnet/pull/3048
+1. (on `release/${RELEASE_NAME}`) Start a topology cluster using the script mentioned at the end of this document.
+1. Create a release testnet page in the wiki at: https://www.notion.so/Testnets-e53255f7003f4c8eae2f1b6644a676e0
+   You may use previous testnet pages as templates. Ensure all started nodes are documented.
+1. Share the links to the release tracking issue, tracking PR and testnet wiki page in the `#release` Element channel.
+   On the `#testing` channel, members are expected to run their own nodes (either AVADO or via their workstation) to participate in the release.
+1. Patches to the release are created via `hotfix/RELEASE_NAME/**` branches.
    Each of these merges will trigger a new release version, and re-build our infrastructure
-   for that version. Upon successfullly testing a release, the release branch may be merged back into
-   `master` by following these steps:
-
-- locally create a merge-back branch based on the release branch
-- merge the latest `master` branch into the merge-back branch, in case of conflicts changes
-  from `master` have preference
-- revert changes in `packages/avado/Dockerfile`
-- revert changes in `packages/avado/docker-compose.yml`
-- revert any chain specific changes
-- bump the package versions to the next preminor version:
-  `yarn workspaces foreach -piv --no-private --topological-dev exec -- npm version preminor --preid=next`
-- revert changes made to Avado configuration files as part of the initial release creation
-  a new release version, and re-build our infrastructure for that version. Upon successfullly
-  testing a release, merge it back to trigger a new pre-release version via our actions.
+   for that version. Upon successfullly testing a release
+1. Once the first release version has been built and is running, the release branch should be merged-back into `master` once to trigger version upgrades on `master`. See the next section for details.
+
+Once the release testing has concluded, or if any signifant amount of patches were applied to the release branch, the release branch should be merged back into `master` again.
+
+#### Release merge-back
+
+1. (on `release/${RELEASE_NAME}`) Create a PR branch off of the release branch: `git checkout -b merge-back-release-${RELEASE_NAME}`
```

Comment:
I wonder if we can create a script for this instead of utilizing human labor here.

Reply:
Definitely, but as mentioned that it out of scope of this PR.